vos/setup.cfg

```diff
@@ -50,7 +50,7 @@ edit_on_github = False
 github_project = opencadc/vostools
 install_requires = html2text>=2016.5.29 cadcutils>=1.1.26 future aenum
 # version should be PEP440 compatible (http://www.python.org/dev/peps/pep-0440)
-version = 0.0.1a
+version = 3.3.a.1
```

Comment:
This is a very big version jump.  How will clients respond to this?

Reply:
`0.0.1a` is a leftover from me testing publishing with GitHub Actions and not wanting to unintentionally publish something recent. It was never out.